src/lib/source-file-converter.ts

```diff
@@ -132,6 +140,49 @@ export class SourceFileConverter {
 
     const arraySymbol = symbols.filter((s) => s.name === 'Array')[0];
     this._arrayMapSymbol = arraySymbol.members!.get('map' as any)!;
+
+    // Get original html, property, and customElement symbols.
+    this.htmlSymbol = this.getExportOfModule('lit-html', 'html');
+    this.propertySymbol = this.getExportOfModule('lit-element/lib/decorators', 'property');
```

Comment:
for my information: do these have to reference the original module, or would it work to do: `this.getExportOfModule('lit-element', 'property')`?

Reply:
It had to be the original module. BUT. You inspired me to look into why.

I discovered that `<module symbol>.exports` returns only exports that originate from the module in question, while `checker.getExportsOfModule(<module symbol>)` returns all exports _including_ re-exported declarations. 🙃

Reply:
I'm gonna use the latter approach so we can do `this.getExportOfModule('lit-element', 'property')`, which looks nice